packages/frontend/src/components/FlowRow/DemoFlowModal.tsx

```diff
@@ -14,7 +14,7 @@ import { Badge, BadgeLeftIcon, Button } from '@opengovsg/design-system-react'
 import demoModalImg from '@/assets/demo-modal.png'
 import {
   DEMO_VIDEOS_MAP,
-  FORMSG_POSTMAN_TEMPLATE,
+  FORMSG_POSTMAN_TEMPLATE_ID,
```

Comment:
will these be moved to the backend?

Reply:
i need this on the frontend for fetching the one and only demo flow template